### DISCUSSION TITLE
nocodb on digitalOcean [Sep 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3822] [Discord Threads]
> 🙋‍♂️ **emre_**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### steps by dstala@discord    
####    
#### 1. Visit https://cloud.digitalocean.com/apps/   
#### 2. Click 'Create App'   
#### 3. Choose Source: Select "Docker Hub"   
#### 4. Repository: type "nocodb/nocodb"   
#### 5. Next page, keep everything default (port number et.al)   
#### 6. Optionally, pick a name for your web-service && Region near you   
#### 7. Select plan, click 'Launch Pro App'   
#### It will take a minute for setting up & provide URL for the application!  
#  
> 🙋‍♂️ **emre_**  
#### Thanks, does it set everything with postgres in this way or sqlite? sorry i am a frontend person, don't know much about docker etc. 🙁  
#  
> 🙋‍♂️ **wkw**  
#### please check out the architecture    
#### https://docs.nocodb.com/#architecture---simple-overview   
####    
#### if NC_DB is not present, we'll create a sqlite for storing the metadata.  
#  
> 🙋‍♂️ **wkw**  
#### if u hv an existing database for storing data, you can specify it when you create a nocodb project. otherwise, a sqlite will be used  
#  
> 💁 **o1lab**  
#### if you want to run it on a droplet emre_@discord : please use docker-compose setup  
#  
> 🙋‍♂️ **emre_**  
#### okay thanks, i think i need to experiment. So how about the last question. would 5$ droplet enough for nocodb or should i get something stronger? at max 50 different people will use it at once.  
#  
> 💁 **o1lab**  
#### yep should be good  
#  
> 💁 **o1lab**  
#### it takes about 120Mb of ram  
#  
> 💁 **o1lab**  
#### for 50 ppl I think it would be 256 Mb will be suffice  
#  
> 🙋‍♂️ **emre_**  
#### perfect thanks alot for the info. btw do you have any info for 1 click heroku setup? how much data and concurrent users can handle the free heroku account?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/955797325366853713)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
